constraints/recursiveCatEvals.js

```diff
@@ -5,18 +5,29 @@ Assumes all nodes have valid and relevant categories
 (i.e., this is designed for prosodic trees and won't give the desired results
 if run on a syntactic tree that contains, e.g., bar levels).
 */
-function isMinimal(node){
-	var cat = node.cat;
+function isMinimal(node, lastCat){
+	//use the lastCat argument as the cat under question if it exists.
+	if(lastCat){
```

Comment:
Please expand on this comment to explain what lastCat and when it would be defined. Or you can put that information in the comment above the function declaration.